tests/integration/email_link_send_test.go

```diff
@@ -0,0 +1,28 @@
+//go:build integration
+
+package integration_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/corbado/corbado-go/pkg/sdk/entity/api"
+	"github.com/corbado/corbado-go/pkg/sdk/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEmailLinkSend(t *testing.T) {
+	sdk := initClient(t)
+	rsp, err := sdk.EmailLinks().Send(context.TODO(), api.EmailLinkSendReq{
+		Email:             createRandomTestEmail(t),
+		Redirect:          "https://some.site.com/authenticate",
+		TemplateName:      util.Ptr("default"),
+		Create:            true,
+		AdditionalPayload: util.Ptr("{}"),
+		ClientInfo:        util.ClientInfo("foobar", "127.0.0.1"),
+	})
+
+	require.NoError(t, err)
+	assert.NotEmpty(t, rsp.Data.EmailLinkID)
```

Comment:
should discuss this in whatsapp: do we want to assert on every field (lots of work but will catch the most bugs) or do we skips this and make the asserts easy aka "no error and 'some' data". maybe check the others, i think stripe is doing the first